sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/SqlScriptingParserSuite.scala

```diff
@@ -224,13 +227,15 @@ class SqlScriptingParserSuite extends SparkFunSuite with SQLHelper {
         |  SELECT a, b, c FROM T;
         |  SELECT * FROM T;
         |END lbl""".stripMargin
-
+    val exception = intercept[SqlScriptingException] {
+      parseScript(sqlScriptText)
+    }
```

Comment:
Is there a reason why we removed this part to be outside of `checkError`? In most existing tests it is inside.

Reply:
SGTM